android/src/main/java/com/raulh82vlc/MoviesFinder/ui/activities/MovieDetailsActivity.java

```diff
@@ -190,13 +191,20 @@ protected void setInitialTitle() {
 
     @Override
     protected void setToolbarInitialisation() {
-        mToolbar.setNavigationIcon(R.drawable.ic_back);
+        //mToolbar.setNavigationIcon(R.drawable.ic_back);
```

Comment:
every code not used and which is not a descriptive comment must be removed, consequently the resource `R.drawable.ic_back` should be removed as well, as now it is never used on behalf of the default icon from the support library